style.scss

```diff
@@ -0,0 +1,139 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,700;1,800;1,900&family=Roboto:wght@100;300;400;500;700;900&display=swap");
+.header {
+    display: flex;
+    border-bottom: 1px solid gray;
+}
+* {
+    margin: 0;
+    padding: 0;
+    font-family: "Poppins", sans-serif;
+}
+.header {
+    .logo {
+        img {
+            width: 80px;
+            height: 40px;
+        }
+    }
+}
+.header {
+    justify-content: space-between;
+    padding-left: 165px;
+    padding-right: 165px;
+    height: 80px;
+    align-items: center;
+}
+.header_content_and_btn {
+    align-items: center;
+    gap: 50px;
+}
+.email {
+    display: flex;
+    gap: 40px;
+    margin-top: 14px;
+    align-items: center;
+}
+.num {
+    @extend .email;
+}
+.email {
+    p {
+        i {
+            color: #86be3f;
+        }
+    }
+}
+.num {
+    p {
+        i {
+            color: #86be3f;
+        }
+    }
+}
+.header_content_and_btn {
+    button {
+        background: #86be3f;
+        border: none;
+        padding: 7px 15px;
+        border-radius: 5px;
+        color: #ffffff;
+    }
+}
+.nav_links {
+    padding: 0 165px;
+    gap: 55px;
+    height: 80px;
+    align-items: center;
+}
+.nav_links {
+    li {
+        list-style: none;
+    }
+}
+.nav_links {
+    li {
+        a {
```

Comment:
px to rem convert, create _variable.scss file and import here, always use generic class for same styles